### PR TITLE
TC-ACE_1.6: Rework test to use TH keys

### LINKED ISSUE #27982: [Groups] TC-ACE-1.6 - `0x000000AC: Internal error` causes failure in YAML
### Reproduction steps

Summary: According to test-plan test ACE-1.6 step 7: `AddGroup` command with groupID set to 0x0104 expected outcome is `no response` but got `failure: src/transport/SessionManager.cpp:194: CHIP Error 0x000000AC: Internal error` when executed.

Diff link: https://github.com/project-chip/connectedhomeip/compare/master...manjunath-grl:connectedhomeip:ACE_Groups_ChipError_Response

Reproduction step:
- Terminal 1:

  1. Build All-cluster-app 
       `scripts/examples/gn_build_example.sh      examples/all-clusters-app/linux/      examples/all-clusters-app/linux/out/all-clusters-app   chip_inet_config_enable_ipv4=false`
  2. Run ./chip-all-clusters-app

- Terminal 2:

  1. Build Chip-tool :
       `gn gen out/debug --args='chip_mdns="platform" chip_inet_config_enable_ipv4=false' 
        ninja -C out/debug`
  2. Commission DUT and TH :
       `./out/debug/chip-tool pairing code 0x12344321 MT:-24J0AFN00KA0648G00`
  3. Run ACE-1.6 test:
       `./scripts/tests/yaml/chiptool.py tests Test_TC_ACE_1_6`

### Bug prevalence

Everytime when executed

### GitHub hash of the SDK that was being used

b6caecf7860072bd4e55be43dc830b5ce990ddef

### Platform

other

### Platform Version(s)

_No response_

### Anything else?

Test-plan:
image
Execution logs:
Screenshot from 2023-07-13 12-03-57
[ACE_Fail_logs.zip](https://github.com/project-chip/connectedhomeip/files/12067019/ACE_Fail_logs.zip)

### PR DESCRIPTION
**Pull request by cecille**
Fixes: https://github.com/project-chip/connectedhomeip/issues/27982

Test plan change: https://github.com/CHIP-Specifications/chip-test-plans/pull/3754

### CODE REVIEW

File: src/app/tests/suites/certification/Test_TC_ACE_1_6.yaml [hunk 1]
```diff
@@ -74,21 +74,45 @@ tests:
                         EpochKey2: "hex:d2d1d2d3d4d5d6d7d8d9dadbdcdddedf",
```
Comment (tehampson) [inline src/app/tests/suites/certification/Test_TC_ACE_1_6.yaml:74]: Is there a change to test-plan spec PR for the changes you are making here?

Will you be removing the old step#1

Comment (cecille) [reply src/app/tests/suites/certification/Test_TC_ACE_1_6.yaml:74]: not yet - waiting to hear back from the QA team about whether this is OK for them. Works for me, but I'm not going to do the TP until they're good.

Need both steps now, so I suppose I should call them step 1a and step 1b

Comment (cecille) [reply src/app/tests/suites/certification/Test_TC_ACE_1_6.yaml:74]: Test plan change here: https://github.com/CHIP-Specifications/chip-test-plans/pull/3754

File: src/app/tests/suites/certification/Test_TC_ACE_1_6.yaml [hunk 2]
```diff
@@ -74,21 +74,45 @@ tests:
...
+      cluster: "Group Key Management"
+      command: "KeySetWrite"
+      arguments:
+          values:
+              - name: GroupKeySet
+                value:
+                    {
+                        GroupKeySetID: 0x01a1,
+                        GroupKeySecurityPolicy: 0,
+                        EpochKey0: "hex:a0d1d2d3d4d5d6d7d8d9dadbdcdddedf",
```
Comment (jmeg-sfy) [inline src/app/tests/suites/certification/Test_TC_ACE_1_6.yaml:94]: i thought we created some macros for those epoch keys.. this is a nit

**Review by jmeg-sfy — APPROVED** [review]

**Review by tehampson — APPROVED** [review]

File: src/app/tests/suites/certification/Test_TC_ACE_1_6.yaml (hunk 2 above)
Comment (cecille) [reply src/app/tests/suites/certification/Test_TC_ACE_1_6.yaml:94]: This one definitely wouldn't have one - I don't think it's used anywhere else.

Comment (cecille) [reply src/app/tests/suites/certification/Test_TC_ACE_1_6.yaml:94]: let me see if I can find the macro and add the other keys

Comment (cecille) [reply src/app/tests/suites/certification/Test_TC_ACE_1_6.yaml:94]: oh, wait, this is the yaml. Probably in the test plan. Does YAML do macros?

Comment (cecille) [reply src/app/tests/suites/certification/Test_TC_ACE_1_6.yaml:94]: I don't think we currently support global macros in the yaml, so I'm going to close this for now. I think this probably relates to the test plan

**Pull request merged**